chart/kubedb-catalog/templates/postgres.yaml

```diff
@@ -313,4 +313,120 @@ spec:
     databasePolicyName: postgres-db
     snapshotterPolicyName: postgres-snapshot
 
+---
+# After 0.11.0
+
+apiVersion: catalog.kubedb.com/v1alpha1
+kind: PostgresVersion
+metadata:
+  name: "9.6-v4"
+  labels:
+    {{- include "kubedb-catalog.labels" . | nindent 4 }}
+spec:
+  version: "9.6"
+  db:
+    image: "{{ .Values.dockerRegistry }}/postgres:9.6-v5"
+  exporter:
+    image: "{{ .Values.dockerRegistry }}/postgres_exporter:v0.4.7"
+  tools:
+    image: "{{ .Values.dockerRegistry }}/postgres-tools:9.6-v3"
+  podSecurityPolicies:
+    databasePolicyName: postgres-db
+    snapshotterPolicyName: postgres-snapshot
+
+---
+apiVersion: catalog.kubedb.com/v1alpha1
+kind: PostgresVersion
+metadata:
+  name: "9.6.7-v4"
```

Comment:
9.6.7-v5 ?
https://github.com/kubedb/postgres/blob/master/hack/docker/postgres/9.6.7/make.sh#L13

Reply:
That's DB crd name. DB image is v5, in sync with the DB image suffix. DB image is ahead by one since 9.6-v1 (same for 9.6.7, and 10.2).